mock/tests/testenvironment

```diff
@@ -14,7 +14,7 @@ outdir=${TOPDIR}/mock-unit-test
 cfgdir=${TOPDIR}/../mock-core-configs/etc/mock
 #MOCKCMD="sudo ./py/mock.py $VERBOSE --resultdir=$outdir --uniqueext=$uniqueext --configdir=$cfgdir -r $testConfig --plugin-option=root_cache:age_check=False $MOCK_EXTRA_ARGS"
 MOCKCMD="mock $VERBOSE --resultdir=$outdir --uniqueext=$uniqueext --configdir=$cfgdir -r $testConfig $MOCK_EXTRA_ARGS"
-MOCKCHAIN="mockchain -r $testConfig"
+MOCKCHAIN="mock -r $testConfig"
```

Comment:
Wouldn't you need `--chain` here?

Reply:
I don't need it, because when you put more than one rpm it will automatically call chain functionality and every test about mockchain uses more than one rpm.